docs/code-examples/box3d_batch.cpp

```diff
@@ -22,7 +22,7 @@ int main() {
                     rr::datatypes::Angle::degrees(30.0f)
                 ),
             })
-            .with_radii(0.025f)
+            .with_radii({0.025f})
```

Comment:
why are the `{}` needed here but not for the `LineStrips3D(linestrip)` case? O.o

Reply:
Afaik the problem is that without the compiler immediately wants to apply `ComponentBatchAdapter<Radius, float>` which doesn't exist (we could implement that though :)). With the {} in place it knows that it first needs to call the Radius constructor and it does though. Frankly I find it bizarre that this works.